conf/configuration.go

```diff
@@ -2,79 +2,76 @@ package conf
 
 import (
 	"os"
-	"strconv"
 	"time"
 
+	"github.com/joho/godotenv"
+	"github.com/kelseyhightower/envconfig"
 	"github.com/netlify/netlify-commons/nconf"
-	"github.com/pkg/errors"
-	"github.com/spf13/cobra"
 )
 
 // ExternalConfiguration holds all config related to external account providers.
 type ExternalConfiguration struct {
-	ClientID    string `json:"client_id"`
+	ClientID    string `json:"client_id" split_words:"true"`
```

Comment:
Yes, because we deserialize all the non-global configuration for instance provisioning/updating.